Formula/openmittsu.rb

```diff
@@ -0,0 +1,33 @@
+class Openmittsu < Formula
+  desc "Cross-platform GPLv2 desktop client for the Threema Messenger App"
+  homepage "https://github.com/blizzard4591/openMittsu"
+  url "https://github.com/blizzard4591/openMittsu/archive/0.9.15.tar.gz"
+  sha256 "7c70de775716f8c01c6f6f4c812fd938f2aea979f6fe4562bb6a4089fa3b9cae"
+  head "https://github.com/blizzard4591/openMittsu.git", :using => :git, :shallow => false
+
+  depends_on "blizzard4591/homebrew-qt5-sqlcipher/qt5-sqlcipher"
```

Comment:
You can't depend on personal taps

Reply:
Thank you for the hint, will add a PR for that as well!